owncloud/owncloud.py

```diff
@@ -1016,6 +1016,28 @@ def search_users(self, user_name):
 
         raise HTTPResponseError(res)
 
+    def get_users(self):
```

Comment:
It looks like the existing method "search_users" already does that but it requires an additional argument.

I suggest you rewire it with "get_users" to avoid duplicate code.
Maybe rewrite `get_users` to call `search_users` without argument, and make `search_users` accept the empty argument.